ui/packages/catalog-ui-search/src/main/webapp/component/search-interactions/search-interactions.view.js

```diff
@@ -13,25 +13,28 @@
  *
  **/
 /*global define*/
-var Marionette = require('marionette');
-var _ = require('underscore');
-var $ = require('jquery');
-var template = require('./search-interactions.hbs');
-var CustomElements = require('js/CustomElements');
-var lightboxInstance = require('component/lightbox/lightbox.view.instance');
-var SearchSettingsDropdownView = require('component/dropdown/search-settings/dropdown.search-settings.view');
-var DropdownModel = require('component/dropdown/dropdown');
-var SearchFormSelectorDropdownView = require('component/dropdown/search-form-selector/dropdown.search-form-selector.view');
-var _merge = require('lodash/merge');
-var ConfirmationView = require('component/confirmation/confirmation.view');
-var user = require('component/singletons/user-instance');
+const Marionette = require('marionette');
+const _ = require('underscore');
+const $ = require('jquery');
+const template = require('./search-interactions.hbs');
+const CustomElements = require('js/CustomElements');
+const lightboxInstance = require('component/lightbox/lightbox.view.instance');
+const SearchSettingsDropdownView = require('component/dropdown/search-settings/dropdown.search-settings.view');
+const DropdownModel = require('component/dropdown/dropdown');
+const SearchFormSelectorDropdownView = require('component/dropdown/search-form-selector/dropdown.search-form-selector.view');
+const _merge = require('lodash/merge');
```

Comment:
Is this being used?

Reply:
I dont see a use of _merge but I used the _.extendOwn Josh recommended in metacard-definitions